server/routes/bookings/bookingsController.ts

```diff
@@ -1,21 +1,28 @@
 import type { RequestHandler } from 'express'
 import { SessionData } from 'express-session'
 import createError from 'http-errors'
-import { VisitService } from '../../services'
-import getPrisonInformation from '../../constants/prisonInformation'
+import { PrisonService, VisitService } from '../../services'
 import { VisitDetails } from '../../services/visitService'
+import { PrisonDto } from '../../data/orchestrationApiTypes'
 
 export default class BookingsController {
-  public constructor(private readonly visitService: VisitService) {}
+  public constructor(
+    private readonly prisonService: PrisonService,
+    private readonly visitService: VisitService,
+  ) {}
 
   public view(type: SessionData['bookings']['type']): RequestHandler {
     return async (req, res, next) => {
       const { booker } = req.session
 
       let visits: VisitDetails[]
+      let prison: PrisonDto
       switch (type) {
         case 'future':
           visits = await this.visitService.getFuturePublicVisits(booker.reference)
+          // making assumption here that all visits[] will be for the same prison
```

Comment:
This can be addressed when the service supports more than one prisoner per booker.